### PR TITLE
Update driver for 0.8 client pravega

### PR DESCRIPTION
**Pull request by dadlex**

### CODE REVIEW

**Comment by veronikaKochugova** [comment]
👍

File: gradle.properties [hunk 1]
```diff
@@ -1,2 +1,2 @@
-pravegaCommit=4bf247f
-pravegaVersion=0.7.0-rc0
+pravegaCommit=355579c
+pravegaVersion=0.8.0-pre-release
```
Comment (AndreyKoltsov1997) [inline gradle.properties]: I think we should change that configuration with respect to Pravega 0.8.0 release information: https://github.com/pravega/pravega/releases/tag/v0.8.0
```
pravegaCommit=e4c436b
pravegaVersion=v0.8.0
```

File: src/main/java/com/emc/mongoose/storage/driver/pravega/PravegaStorageDriver.java [hunk 2]
```diff
@@ -364,6 +363,7 @@ public PravegaStorageDriver(
 			this.transactionMode = eventConfig.boolVal("transaction");
 		} else {
 			this.transactionMode = false;
+			throw new IllegalConfigurationException("ByteStreams are not currently supported. Contact the dev team.");
```
Comment (AndreyKoltsov1997) [inline src/main/java/com/emc/mongoose/storage/driver/pravega/PravegaStorageDriver.java:366]: Please, add this information into README.md's `Byte Stream` section as well: https://github.com/emc-mongoose/mongoose-storage-driver-pravega/tree/PRAVEGA-101-update-for-0.8-pravega#52-byte-stream-operations

**Review by AndreyKoltsov1997 — CHANGES_REQUESTED** [review]

File: gradle.properties (hunk 1 above)
Comment (dadlex) [reply gradle.properties]: fair enough

**Review by AndreyKoltsov1997 — APPROVED** [review]
LGTM assuming Pravega's tags have been commented out intentionally.

**Pull request merged**